Traceback (most recent call last):
  File "/home/umberto/.local/lib/python3.8/site-packages/jupyter_cache/executors/utils.py", line 51, in single_nb_execution
    executenb(
  File "/home/umberto/.local/lib/python3.8/site-packages/nbclient/client.py", line 1204, in execute
    return NotebookClient(nb=nb, resources=resources, km=km, **kwargs).execute()
  File "/home/umberto/.local/lib/python3.8/site-packages/nbclient/util.py", line 84, in wrapped
    return just_run(coro(*args, **kwargs))
  File "/home/umberto/.local/lib/python3.8/site-packages/nbclient/util.py", line 62, in just_run
    return loop.run_until_complete(coro)
  File "/usr/lib/python3.8/asyncio/base_events.py", line 616, in run_until_complete
    return future.result()
  File "/home/umberto/.local/lib/python3.8/site-packages/nbclient/client.py", line 663, in async_execute
    await self.async_execute_cell(
  File "/home/umberto/.local/lib/python3.8/site-packages/nbclient/client.py", line 965, in async_execute_cell
    await self._check_raise_for_error(cell, cell_index, exec_reply)
  File "/home/umberto/.local/lib/python3.8/site-packages/nbclient/client.py", line 862, in _check_raise_for_error
    raise CellExecutionError.from_cell_and_msg(cell, exec_reply_content)
nbclient.exceptions.CellExecutionError: An error occurred while executing the following cell:
------------------
fig, ax = plt.subplots(figsize=(10,6)) 
plt.plot(x, y ,label='Original signal',linewidth=2.0)
plt.xlabel('$t [s]$',fontsize=24)
plt.ylabel('$x [mm]$',fontsize=24)
plt.legend(bbox_to_anchor=(1.02, 1.05), fontsize=20)
plt.tick_params(axis='both', which='major', labelsize=15)     
plt.show()
------------------

[0;31m---------------------------------------------------------------------------[0m
[0;31mNameError[0m                                 Traceback (most recent call last)
[0;32m/tmp/ipykernel_27082/443173906.py[0m in [0;36m<module>[0;34m[0m
[1;32m      1[0m [0mfig[0m[0;34m,[0m [0max[0m [0;34m=[0m [0mplt[0m[0;34m.[0m[0msubplots[0m[0;34m([0m[0mfigsize[0m[0;34m=[0m[0;34m([0m[0;36m10[0m[0;34m,[0m[0;36m6[0m[0;34m)[0m[0;34m)[0m[0;34m[0m[0;34m[0m[0m
[0;32m----> 2[0;31m [0mplt[0m[0;34m.[0m[0mplot[0m[0;34m([0m[0mx[0m[0;34m,[0m [0my[0m [0;34m,[0m[0mlabel[0m[0;34m=[0m[0;34m'Original signal'[0m[0;34m,[0m[0mlinewidth[0m[0;34m=[0m[0;36m2.0[0m[0;34m)[0m[0;34m[0m[0;34m[0m[0m
[0m[1;32m      3[0m [0mplt[0m[0;34m.[0m[0mxlabel[0m[0;34m([0m[0;34m'$t [s]$'[0m[0;34m,[0m[0mfontsize[0m[0;34m=[0m[0;36m24[0m[0;34m)[0m[0;34m[0m[0;34m[0m[0m
[1;32m      4[0m [0mplt[0m[0;34m.[0m[0mylabel[0m[0;34m([0m[0;34m'$x [mm]$'[0m[0;34m,[0m[0mfontsize[0m[0;34m=[0m[0;36m24[0m[0;34m)[0m[0;34m[0m[0;34m[0m[0m
[1;32m      5[0m [0mplt[0m[0;34m.[0m[0mlegend[0m[0;34m([0m[0mbbox_to_anchor[0m[0;34m=[0m[0;34m([0m[0;36m1.02[0m[0;34m,[0m [0;36m1.05[0m[0;34m)[0m[0;34m,[0m [0mfontsize[0m[0;34m=[0m[0;36m20[0m[0;34m)[0m[0;34m[0m[0;34m[0m[0m

[0;31mNameError[0m: name 'x' is not defined
NameError: name 'x' is not defined

